mtags/src/main/scala-2/scala/meta/internal/mtags/MtagsEnrichments.scala

```diff
@@ -133,8 +133,13 @@ trait MtagsEnrichments extends CommonMtagsEnrichments {
         URLDecoder.decode(value, "UTF-8").toAbsolutePath(followSymlink)
       else if (value.toUpperCase.startsWith("JAR:FILE%3A"))
         URLDecoder.decode(value, "UTF-8").toAbsolutePath(followSymlink)
-      else
-        URI.create(value.stripPrefix("metals:")).toAbsolutePath(followSymlink)
+      else if (value.toUpperCase.startsWith("JAR"))
+        URI.create(value).toAbsolutePath(followSymlink)
```

Comment:
We do not want to overcode JARs, right @Arthurm1?

Reply:
Good question - I don't think so.
I'm trying to move away from `jar` URIs in https://github.com/scalameta/metals/pull/3750 so that only Metals will deal with `jar` URIs and only in a few specific classes and the client won't encounter them at all so won't mess with the encoded value.